cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -2522,15 +2526,15 @@ impl Inst {
             &Inst::TrapIf { ref kind, .. } => match kind {
                 &CondBrKind::Zero(reg) => {
                     let reg = pretty_print_reg(reg, allocs);
-                    format!("cbnz {}, 8 ; udf", reg)
+                    format!("cbz {}, #trap", reg)
```

Comment:
Could we print the `trap_code` in our disassembly, maybe in a comment? It's visible in the Capstone disassembly as part of the trap pool, but it's hard to see which branch goes to which element of the trap pool, and in our disassembly the trap pool isn't printed at all.